apps/wakunode2/external_config.nim

```diff
@@ -225,10 +225,13 @@ type
       defaultValue: false
       name: "keep-alive" }: bool
 
-    topics* {.
-      desc: "Default topic to subscribe to. Argument may be repeated."
-      defaultValue: @["/waku/2/default-waku/proto"]
-      name: "topic" .}: seq[string]
+    pubsubTopics* {.
+      desc: "Default pubsub topic to subscribe to. Argument may be repeated."
+      name: "pubsub-topic" .}: seq[string]
```

Comment:
I wouldn't remove the default here for now. For some applications this will be a breaking change (for example, I think DappNode may assume subscription to default topic).